app/app.go

```diff
@@ -301,37 +318,24 @@ func New(
 	bApp.SetVersion(version.Version)
 	bApp.SetInterfaceRegistry(interfaceRegistry)
 
-	var keys map[string]*sdk.KVStoreKey
+	storeKeys := []string{
+		authtypes.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
+		minttypes.StoreKey, distrtypes.StoreKey, slashingtypes.StoreKey,
+		govtypes.StoreKey, paramstypes.StoreKey, upgradetypes.StoreKey,
+		evidencetypes.StoreKey, capabilitytypes.StoreKey,
+		feegrant.StoreKey, authzkeeper.StoreKey,
+		// ibc keys
+		ibchost.StoreKey, ibctransfertypes.StoreKey,
+		icacontrollertypes.StoreKey,
+		// ethermint keys
+		evmtypes.StoreKey, feemarkettypes.StoreKey,
+		// this line is used by starport scaffolding # stargate/app/storeKey
+		cronostypes.StoreKey,
+	}
 	if experimental {
-		keys = sdk.NewKVStoreKeys(
-			authtypes.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
-			minttypes.StoreKey, distrtypes.StoreKey, slashingtypes.StoreKey,
-			govtypes.StoreKey, paramstypes.StoreKey, upgradetypes.StoreKey,
-			evidencetypes.StoreKey, capabilitytypes.StoreKey,
-			feegrant.StoreKey, authzkeeper.StoreKey,
-			// ibc keys
-			ibchost.StoreKey, ibctransfertypes.StoreKey,
-			// ethermint keys
-			evmtypes.StoreKey, feemarkettypes.StoreKey,
-			gravitytypes.StoreKey,
-			// this line is used by starport scaffolding # stargate/app/storeKey
-			cronostypes.StoreKey,
-		)
-	} else {
-		keys = sdk.NewKVStoreKeys(
-			authtypes.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
-			minttypes.StoreKey, distrtypes.StoreKey, slashingtypes.StoreKey,
-			govtypes.StoreKey, paramstypes.StoreKey, upgradetypes.StoreKey,
-			evidencetypes.StoreKey, capabilitytypes.StoreKey,
-			feegrant.StoreKey, authzkeeper.StoreKey,
-			// ibc keys
-			ibchost.StoreKey, ibctransfertypes.StoreKey,
-			// ethermint keys
-			evmtypes.StoreKey, feemarkettypes.StoreKey,
-			// this line is used by starport scaffolding # stargate/app/storeKey
-			cronostypes.StoreKey,
-		)
+		storeKeys = append(storeKeys, gravitytypes.StoreKey)
```

Comment:
the order of store keys doesn't matter I guess? (before gravity was before cronos)

Reply:
yes, the `NewKVStoreKeys` returns a map.